packages/core/contracts/bot-helpers/atomic-liquidator/ReserveCurrencyLiquidator.sol

```diff
@@ -39,44 +42,64 @@ contract ReserveCurrencyLiquidator {
         address financialContract,
         address reserveCurrency,
         address liquidatedSponsor,
-        FixedPoint.Unsigned calldata maxReserverTokenSpent,
+        FixedPoint.Unsigned calldata maxReserveTokenSpent,
         FixedPoint.Unsigned calldata minCollateralPerTokenLiquidated,
         FixedPoint.Unsigned calldata maxCollateralPerTokenLiquidated,
         FixedPoint.Unsigned calldata maxTokensToLiquidate,
         uint256 deadline
     ) public {
```

Comment:
I thought about reentrancy vulnerabilities here and I don't think there's any issue since this contract doesn't store any internal state, but just want to make sure you've thought about that too and perhaps you add a reentrancy guard if its a concern.

Reply:
ye, not too worried about it. this contract is called from a DSProxy and so the context of state is the DSProxy storage. This contract should never hold funds either.